GATEWAY/src/pages/virtual-tiles/virtual-tiles.ts

```diff
@@ -94,6 +94,13 @@ export class VirtualTilesPage {
     }
   }
 
+  unpairTile = (virtualTile: VirtualTile): void => {
```

Comment:
Can the tilesApi function be called from the HTML? And it updates the list so it shows that is connected without refreshing.